component-library/component-lib/src/lib/tabs/tabs.component.ts

```diff
@@ -0,0 +1,57 @@
+import { EventEmitter, Input, Output } from '@angular/core';
+import { AfterViewInit } from '@angular/core';
+import { ChangeDetectorRef } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { DSSizes } from 'component-lib/src/shared/constants/jl-components/jl-components.constants/jl-components.constants';
+export interface ITabNavConfig {
+  id: string;
+  tab?: ITabConfig[];
+  size?: keyof typeof DSSizes;
+};
+export interface ITabConfig {
+  id?: string,
+  title: string,
+  value?: string
+}
+@Component({
+  selector: 'lib-tabs',
+  templateUrl: './tabs.component.html',
+  styleUrls: ['./tabs.component.css']
```

Comment:
```suggestion
```
Suggest remove this and the empty css file.

Reply:
Good catch! 
Done, thanks! :)